src/dos/dos_programs.cpp

```diff
@@ -1783,6 +1783,29 @@ void DOS_SetupPrograms(void) {
 	        "  \033[32;1mimgmount\033[0m \033[37;1ma\033[0m \033[36;1mfloppy1.img floppy2.img floppy3.img\033[0m -t floppy\n"
 	        "  \033[32;1mimgmount\033[0m \033[37;1mc\033[0m \033[36;1m~/dos/c_drive.img\033[0m -t hdd\n"
 	        "  \033[32;1mimgmount\033[0m \033[37;1mc\033[0m \033[36;1mbootable.img\033[0m -t hdd -fs none -size 512,63,32,1023\n");
+	
+	MSG_Add("SHELL_CMD_MOUNT_HELP",
+	        "connects physical folders and drives to dosbox-staging virtual drives.\n");
+			
+	MSG_Add("SHELL_CMD_MOUNT_HELP_LONG"
```

Comment:
`MSG_Add("SHELL_CMD_MOUNT_HELP_LONG",`

(add a comma at the end, similar to `MSG_Add("SHELL_CMD_MOUNT_HELP",`